lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -1393,12 +1406,15 @@ static llvm::Expected<ParsedExpression> ParseAndImport(
     llvm::SmallVector<SwiftASTManipulator::VariableInfo, 5> local_variables;
 
     if (local_context_is_swift) {
-      AddRequiredAliases(sc.block, stack_frame_sp, *swift_ast_context,
-                         *code_manipulator, options.GetUseDynamic());
+      if (!AddRequiredAliases(sc.block, stack_frame_sp, *swift_ast_context,
+                              *code_manipulator, options.GetUseDynamic()))
+        return make_error<StringError>(inconvertibleErrorCode(),
```

Comment:
Since we turn this into a string error, maybe we should form a more specific error (or error message) in AddRequiredAliases() and return that here instead of the generic "Could not add required aliases." message?